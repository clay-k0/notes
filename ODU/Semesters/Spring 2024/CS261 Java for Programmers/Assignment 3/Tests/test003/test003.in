6
A 10
B 10 A
C 90 B
D 30
E 75 D A
F 30 D E
