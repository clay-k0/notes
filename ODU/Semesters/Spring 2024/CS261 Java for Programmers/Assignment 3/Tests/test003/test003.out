245:  A:10  B:10  C:90  D:30  E:75  F:30
